test/README.md

```diff
@@ -156,6 +156,10 @@ go test -v -count=1 -tags=e2e -timeout=20m ./test
 go test -v -count=1 -tags=e2e -timeout=20m ./test --kubeconfig ~/special/kubeconfig --cluster myspecialcluster
 ```
 
+If tests are applied to the cluster with hardware architecture different to the base one
+(for instance `go test` starts on amd64 architecture and `--kubeconfig` points to s390x Kubernetes cluster),
+use `TEST_RUNTIME_ARCH` environment variable to specify the target hardware architecture.
```

Comment:
Could we add an example here of what valid TEST_RUNTIME_ARCH values are?

Reply:
done